### ISSUE TITLE
Actors not work with dapr init --slim

### ISSUE THREAD

**Issue by kinosang** [dapr/cli#641]
## Expected Behavior

Trying to run the .Net Actor example.

## Actual Behavior

In `DemoActor` run `dapr run --app-id myapp --app-port 5000 --dapr-http-port 3500 dotnet run`
<details>
  <summary>logs</summary>
  
```
ℹ️  Starting Dapr with id myapp. HTTP Port: 3500. gRPC Port: 62014
INFO[0000] starting Dapr Runtime -- version 1.0.1 -- commit 45bc40d  app_id=myapp instance=ChinoChgsMacPro.joy scope=dapr.runtime type=log ver=1.0.1
INFO[0000] log level set to: info                        app_id=myapp instance=ChinoChgsMacPro.joy scope=dapr.runtime type=log ver=1.0.1
INFO[0000] metrics server started on :62015/             app_id=myapp instance=ChinoChgsMacPro.joy scope=dapr.metrics type=log ver=1.0.1
INFO[0000] standalone mode configured                    app_id=myapp instance=ChinoChgsMacPro.joy scope=dapr.runtime type=log ver=1.0.1
INFO[0000] app id: myapp                                 app_id=myapp instance=ChinoChgsMacPro.joy scope=dapr.runtime type=log ver=1.0.1
INFO[0000] mTLS is disabled. Skipping certificate request and tls validation  app_id=myapp instance=ChinoChgsMacPro.joy scope=dapr.runtime type=log ver=1.0.1
INFO[0000] local service entry announced: myapp -> 10.0.0.245:62019  app_id=myapp instance=ChinoChgsMacPro.joy scope=dapr.contrib type=log ver=1.0.1
INFO[0000] Initialized name resolution to standalone     app_id=myapp instance=ChinoChgsMacPro.joy scope=dapr.runtime type=log ver=1.0.1
INFO[0000] component loaded. name: messagebus, type: pubsub.redis/v1  app_id=myapp instance=ChinoChgsMacPro.joy scope=dapr.runtime type=log ver=1.0.1
INFO[0000] waiting for all outstanding components to be processed  app_id=myapp instance=ChinoChgsMacPro.joy scope=dapr.runtime type=log ver=1.0.1
INFO[0000] component loaded. name: statestore, type: state.redis/v1  app_id=myapp instance=ChinoChgsMacPro.joy scope=dapr.runtime type=log ver=1.0.1
INFO[0000] all outstanding components processed          app_id=myapp instance=ChinoChgsMacPro.joy scope=dapr.runtime type=log ver=1.0.1
INFO[0000] enabled gRPC tracing middleware               app_id=myapp instance=ChinoChgsMacPro.joy scope=dapr.runtime.grpc.api type=log ver=1.0.1
INFO[0000] enabled gRPC metrics middleware               app_id=myapp instance=ChinoChgsMacPro.joy scope=dapr.runtime.grpc.api type=log ver=1.0.1
INFO[0000] API gRPC server is running on port 62014      app_id=myapp instance=ChinoChgsMacPro.joy scope=dapr.runtime type=log ver=1.0.1
INFO[0000] enabled metrics http middleware               app_id=myapp instance=ChinoChgsMacPro.joy scope=dapr.runtime.http type=log ver=1.0.1
INFO[0000] enabled tracing http middleware               app_id=myapp instance=ChinoChgsMacPro.joy scope=dapr.runtime.http type=log ver=1.0.1
INFO[0000] http server is running on port 3500           app_id=myapp instance=ChinoChgsMacPro.joy scope=dapr.runtime type=log ver=1.0.1
INFO[0000] The request body size parameter is: 4         app_id=myapp instance=ChinoChgsMacPro.joy scope=dapr.runtime type=log ver=1.0.1
INFO[0000] enabled gRPC tracing middleware               app_id=myapp instance=ChinoChgsMacPro.joy scope=dapr.runtime.grpc.internal type=log ver=1.0.1
INFO[0000] enabled gRPC metrics middleware               app_id=myapp instance=ChinoChgsMacPro.joy scope=dapr.runtime.grpc.internal type=log ver=1.0.1
INFO[0000] internal gRPC server is running on port 62019  app_id=myapp instance=ChinoChgsMacPro.joy scope=dapr.runtime type=log ver=1.0.1
INFO[0000] application protocol: http. waiting on port 5000.  This will block until the app is listening on that port.  app_id=myapp instance=ChinoChgsMacPro.joy scope=dapr.runtime type=log ver=1.0.1
ℹ️  Updating metadata for app command: dotnet run
✅  You're up and running! Both Dapr and your app logs will appear here.

== APP == info: Microsoft.Hosting.Lifetime[0]

== APP ==       Now listening on: http://localhost:5000

== APP == info: Microsoft.Hosting.Lifetime[0]

== APP ==       Application started. Press Ctrl+C to shut down.

== APP == info: Microsoft.Hosting.Lifetime[0]

== APP ==       Hosting environment: Development

== APP == info: Microsoft.Hosting.Lifetime[0]

== APP ==       Content root path: /Users/chino/Development/Website/BitCoinary.io/dotnet-sdk/examples/Actor/DemoActor

INFO[0002] application discovered on port 5000           app_id=myapp instance=ChinoChgsMacPro.joy scope=dapr.runtime type=log ver=1.0.1
== APP == info: Microsoft.AspNetCore.Hosting.Diagnostics[1]

== APP ==       Request starting HTTP/1.1 GET http://127.0.0.1:5000/dapr/config application/json

== APP == info: Microsoft.AspNetCore.Routing.EndpointMiddleware[0]

== APP ==       Executing endpoint 'Dapr Actors Config'

== APP == info: Microsoft.AspNetCore.Routing.EndpointMiddleware[1]

== APP ==       Executed endpoint 'Dapr Actors Config'

== APP == info: Microsoft.AspNetCore.Hosting.Diagnostics[2]

== APP ==       Request finished in 24.5804ms 200 application/json

INFO[0002] application configuration loaded              app_id=myapp instance=ChinoChgsMacPro.joy scope=dapr.runtime type=log ver=1.0.1
INFO[0002] actor runtime started. actor idle timeout: 1h0m0s. actor scan interval: 30s  app_id=myapp instance=ChinoChgsMacPro.joy scope=dapr.runtime.actor type=log ver=1.0.1
== APP == info: Microsoft.AspNetCore.Hosting.Diagnostics[1]

== APP ==       Request starting HTTP/1.1 GET http://127.0.0.1:5000/dapr/subscribe application/json

== APP == info: Microsoft.AspNetCore.Hosting.Diagnostics[2]

== APP ==       Request finished in 0.726ms 404

INFO[0002] dapr initialized. Status: Running. Init Elapsed 2674.4529999999995ms  app_id=myapp instance=ChinoChgsMacPro.joy scope=dapr.runtime type=log ver=1.0.1
== APP == info: Microsoft.AspNetCore.Hosting.Diagnostics[1]

== APP ==       Request starting HTTP/1.1 GET http://127.0.0.1:5000/healthz

== APP == info: Microsoft.AspNetCore.Routing.EndpointMiddleware[0]

== APP ==       Executing endpoint 'Dapr Actors Health Check'

== APP == info: Microsoft.AspNetCore.Routing.EndpointMiddleware[1]

== APP ==       Executed endpoint 'Dapr Actors Health Check'

== APP == info: Microsoft.AspNetCore.Hosting.Diagnostics[2]

== APP ==       Request finished in 4.8667ms 200 text/plain
```
</details>

In `ActorClient` run `dotnet run`
```
Making call using actor proxy to save data.
Unhandled exception. Dapr.DaprApiException: error invoke actor method: error finding address for actor type DemoActor with id abc
```

```
$ curl -X POST http://127.0.0.1:3500/v1.0/actors/DemoActor/abc/method/GetData
{"errorCode":"ERR_ACTOR_INVOKE_METHOD","message":"error invoke actor method: error finding address for actor type DemoActor with id abc"}%
```

![screenshot](https://user-images.githubusercontent.com/4987102/112112883-0e67e080-8bf1-11eb-91f8-583a801c122c.png)

## Steps to Reproduce the Problem

Already setup the components and run `placement`.

<details>
  <summary>`~/.dapr/components/statestore.yaml`</summary>

```
apiVersion: dapr.io/v1alpha1
kind: Component
metadata:
  name: statestore
spec:
  type: state.redis
  version: v1
  metadata:
  - name: redisHost
    value: localhost:6379
  - name: redisPassword
    value: ""
  - name: actorStateStore
    value: "true"
```
</details>

<details>
  <summary>`~/.dapr/components/pubsub.yaml`</summary>

```
apiVersion: dapr.io/v1alpha1
kind: Component
metadata:
  name: messagebus
spec:
  type: pubsub.redis
  version: v1
  metadata:
  - name: redisHost
    value: localhost:6379
  - name: redisPassword
    value: ""
```
</details>

<details>
  <summary>run `placement`</summary>
  
```
INFO[0000] starting Dapr Placement Service -- version 1.0.1 -- commit 45bc40d  instance=ChinoChgsMacPro.joy scope=dapr.placement type=log ver=1.0.1
INFO[0000] log level set to: info                        instance=ChinoChgsMacPro.joy scope=dapr.placement type=log ver=1.0.1
INFO[0000] metrics server started on :9090/              instance=ChinoChgsMacPro.joy scope=dapr.metrics type=log ver=1.0.1
INFO[0000] Raft server is starting on 127.0.0.1:8201...  instance=ChinoChgsMacPro.joy scope=dapr.placement.raft type=log ver=1.0.1
INFO[0000] placement service started on port 50005       instance=ChinoChgsMacPro.joy scope=dapr.placement type=log ver=1.0.1
INFO[0000] Healthz server is listening on :8080          instance=ChinoChgsMacPro.joy scope=dapr.placement type=log ver=1.0.1
INFO[0001] cluster leadership acquired                   instance=ChinoChgsMacPro.joy scope=dapr.placement type=log ver=1.0.1
INFO[0001] leader is established.                        instance=ChinoChgsMacPro.joy scope=dapr.placement type=log ver=1.0.1
```
</details>

**Comment by kinosang** [comment]
ref dapr/dotnet-sdk#380 #452

**Comment by wcs1only** [comment]
If you run placement with `placement --port 6050` does it work correctly?

**Comment by mukundansundar** [comment]
> What OS are you on. If you are on Windows, if you run placement with `placement --port 6050` does it work correctly?

@kinosang Is there any update on the above comment?

**Comment by yaron2** [comment]
Ping @kinosang

**Comment by kinosang** [comment]
@yaron2 sorry for the long delay, I'm no longer trying the dapr framework. Thx for your follow-up.

But I'll have a test asap.

**Comment by pravinpushkar** [comment]
@kinosang  @mukundansundar  I could not reproduce the issue, below is output from ActorClient - 
```
Making call using actor proxy to save data.
Making call using actor proxy to get data.
Received data is PropertyA: ValueA, PropertyB: ValueB.
Making calls to an actor method which has no argument and no return type.
Got Incorrect Exception from actor method invocation. Exception Dapr.Actors.ActorInvokeException: The method or operation is not implemented.
Making calls without Remoting.
Registering the timer and reminder
Waiting so the timer and reminder can be triggered
Making call using actor proxy to get data after timer and reminder triggered
Received data is PropertyA: Timer triggered at '02022-07-16T14:33:9', PropertyB: Reminder triggered at '2022-07-16T14:33:08'.
Deregistering timer. Timers would any way stop if the actor is deactivated as part of Dapr garbage collection.
Deregistering reminder. Reminders are durable and would not stop until an explicit deregistration or the actor is deleted.
Creating a Bank Actor
Balance for account 'c5a58e9f-e522-4b25-a9d8-5830e65b8d20' is '¤100.00'.
Withdrawing '¤10.00'...
Balance for account 'c5a58e9f-e522-4b25-a9d8-5830e65b8d20' is '¤90.00'.
Withdrawing '¤10.00'...
Balance for account 'c5a58e9f-e522-4b25-a9d8-5830e65b8d20' is '¤80.00'.
Withdrawing '¤10.00'...
Balance for account 'c5a58e9f-e522-4b25-a9d8-5830e65b8d20' is '¤70.00'.
Withdrawing '¤10.00'...
Balance for account 'c5a58e9f-e522-4b25-a9d8-5830e65b8d20' is '¤60.00'.
Withdrawing '¤10.00'...
Balance for account 'c5a58e9f-e522-4b25-a9d8-5830e65b8d20' is '¤50.00'.
Withdrawing '¤10.00'...
Balance for account 'c5a58e9f-e522-4b25-a9d8-5830e65b8d20' is '¤40.00'.
Withdrawing '¤10.00'...
Balance for account 'c5a58e9f-e522-4b25-a9d8-5830e65b8d20' is '¤30.00'.
Withdrawing '¤10.00'...
Balance for account 'c5a58e9f-e522-4b25-a9d8-5830e65b8d20' is '¤20.00'.
Withdrawing '¤10.00'...
Balance for account 'c5a58e9f-e522-4b25-a9d8-5830e65b8d20' is '¤10.00'.
Withdrawing '¤10.00'...
Balance for account 'c5a58e9f-e522-4b25-a9d8-5830e65b8d20' is '¤0.00'.
Withdrawing '¤10.00'...
Balance for account 'c5a58e9f-e522-4b25-a9d8-5830e65b8d20' is '(¤10.00)'.
Withdrawing '¤10.00'...
Balance for account 'c5a58e9f-e522-4b25-a9d8-5830e65b8d20' is '(¤20.00)'.
Withdrawing '¤10.00'...
Balance for account 'c5a58e9f-e522-4b25-a9d8-5830e65b8d20' is '(¤30.00)'.
Withdrawing '¤10.00'...
Balance for account 'c5a58e9f-e522-4b25-a9d8-5830e65b8d20' is '(¤40.00)'.
Withdrawing '¤10.00'...
Balance for account 'c5a58e9f-e522-4b25-a9d8-5830e65b8d20' is '(¤50.00)'.
Withdrawing '¤10.00'...
Overdraft: Remote Actor Method Exception
➜  ActorClient git:(master) ✗ curl -X POST http://127.0.0.1:3500/v1.0/actors/DemoActor/abc/method/GetData
{"propertyA":"Timer triggered at \u002702022-07-16T14:33:9\u0027","propertyB":"Reminder triggered at \u00272022-07-16T14:33:08\u0027"}%
```

**Comment by mukundansundar** [comment]
Closing this issue as it's not reproducible. Please create a new issue if this occurs again.
